webapp/src/main/java/org/apache/atlas/web/filters/AtlasXSSPreventionFilter.java

```diff
@@ -0,0 +1,208 @@
+package org.apache.atlas.web.filters;
+
+import org.apache.atlas.AtlasConfiguration;
+import org.apache.atlas.web.util.CachedBodyHttpServletRequest;
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang3.StringEscapeUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.owasp.html.HtmlPolicyBuilder;
+import org.owasp.html.PolicyFactory;
+import org.owasp.html.Sanitizers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import com.google.common.base.Predicate;
+
+import java.util.regex.Pattern;
+
+@Component
+public class AtlasXSSPreventionFilter implements Filter {
+
+    private static final Logger LOG = LoggerFactory.getLogger(AtlasXSSPreventionFilter.class);
+    private static Pattern pattern;
+    private PolicyFactory policy;
+    private static final String MASK_STRING = "##ATLAN##";
```

Comment:
What is the use of this?

Reply:
Understood

Reply:
Yeah, it is there just to filter map, array and struct